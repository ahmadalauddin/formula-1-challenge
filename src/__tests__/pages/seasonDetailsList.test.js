import { shallow } from "enzyme";

import SeasonDetailsList from "../../pages/seasonDeatilsList";

let wrapper;

beforeEach(() => {
  wrapper = shallow(<SeasonDetailsList />);
});

it("Checking snapshot", () => {
  expect(wrapper).toMatchSnapshot();
});

it("Checking if padding is applied on top", () => {
  expect(wrapper.exists(".pt-88")).toEqual(true);
});
