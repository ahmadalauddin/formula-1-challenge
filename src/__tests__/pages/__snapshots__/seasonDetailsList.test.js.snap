// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checking snapshot 1`] = `
<div>
  <div
    className="d-flex flex-column pt-88"
  >
    <div
      className="d-flex flex-row justify-content-between"
    >
      <h3
        className="ml-16"
      >
        Season Details - 
      </h3>
      <Link
        className="btn btn-outline-secondary"
        to="/"
        type="button"
      >
        <Memo(ForwardRef(HomeIcon)) />
      </Link>
    </div>
    <div
      className="mt-4"
    >
      <SeasonDetails
        className="mt-3"
      />
    </div>
  </div>
</div>
`;
